packages/app-project/src/components/ProjectHeader/ProjectHeader.js

```diff
@@ -23,7 +23,7 @@ function ProjectHeader (props) {
   const { className, navLinks, projectHomeLink, size, title } = props
 
   return (
-    <StyledBox className={className}>
+    <StyledBox as='header' className={className} aria-label={counterpart('ProjectHeader.ariaLabel')}>
```

Comment:
Why is this labelled? I thought the project had a HTML heading already?

Reply:
There's the site header, and the project header, so it's there to differentiate them

Reply:
Yes, but they have headings already, right?

Reply:
This is what I was thinking of:
>That said, the header and footer elements are exceptions to the rule. These elements should only be exposed to screen readers when they are children of the body element. In other words screen readers should not be aware of multiple headers/footers on a page - so providing extended labels is not necessary.
https://accessibility.blog.gov.uk/2016/05/27/using-navigation-landmarks/#comment-2273

Reply:
Site header _doesn't_ have a heading

Reply:
Ok, but labelled headers aren't supported. Maybe have just one header landmark on the page, around the page heading?

Reply:
From further testing / research:

- I can't see anything in the WCAG docs that say you _can't_ use a label on a header, their examples include labels on `div`s and whatnot;
- I was following [your suggested solution](https://github.com/zooniverse/front-end-monorepo/issues/671#issuecomment-482034433), which seemed reasonable?
-  I've just tested it in VoiceOver, and it looks like labelled headers are supported:

     ![2019-06-18 10 43 48](https://user-images.githubusercontent.com/2725841/59671915-6f5dbb80-91b6-11e9-907f-615c1eb7be55.jpg)

    (Sorry for the crappy photo, I couldn't get screenshots working at the same time as VO.) Are there screenreaders that don't, maybe?
- To prevent duplicate words, I'm going to remove them from the `aria-label`s, e.g. `<nav aria-label="Project" />` should produce `Project navigation` in a screenreader, instead of `Project navigation navigation` - is that cool with you?
- I have a healthy respect for the number of shortcuts used by those using assistive tech

Reply:
My suggestion was wrong, sorry. Reading that discussion above, we're marking up the page banner and my understanding is there should be only one banner.

Reply:
>We have a lot of uppercase links, which are both shouty and can apparently be read as abbreviations in some cases? I'll go through and add labels

Please don't do this.

Reply:
Okay

Reply:
> Please don't do this.

Any references for why not?

Reply:
Because those links already have text, and the uppercase is CSS right? Don't add extra markup because VoiceOver happens to be the IE of screenreaders. I don't think we should be duplicating existing link text.

Reply:
They do, and it is. It doesn't seem like a lot to add (especially since presumably Apple devices _only_ have VO?), but if you're confident that other screenreaders won't render it the same way, then I'll defer to your experience

Reply:
The most common OS/screenreader combinations are probably still Windows/JAWS and Windows/NVDA.